src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/hmi/rc_on_interior_vehicle_data_notification.cc

```diff
@@ -0,0 +1,44 @@
+#include "rc_rpc_plugin/rc_module_constants.h"
+#include "rc_rpc_plugin/commands/hmi/rc_on_interior_vehicle_data_notification.h"
+#include "utils/macro.h"
+
+namespace rc_rpc_plugin {
+namespace commands {
+
+RCOnInteriorVehicleDataNotification::RCOnInteriorVehicleDataNotification(
+    const app_mngr::commands::MessageSharedPtr& message,
+    app_mngr::ApplicationManager& application_manager,
+    app_mngr::rpc_service::RPCService& rpc_service,
+    app_mngr::HMICapabilities& hmi_capabilities,
+    policy::PolicyHandlerInterface& policy_handle,
+    ResourceAllocationManager& resource_allocation_manager)
+    : application_manager::commands::NotificationFromHMI(message,
+                                                         application_manager,
+                                                         rpc_service,
+                                                         hmi_capabilities,
+                                                         policy_handle) {
+  UNUSED(resource_allocation_manager);
+}
+
+RCOnInteriorVehicleDataNotification::~RCOnInteriorVehicleDataNotification() {}
+
+void RCOnInteriorVehicleDataNotification::Run() {
+  LOG4CXX_AUTO_TRACE(logger_);
+  (*message_)[app_mngr::strings::params][app_mngr::strings::function_id] =
+      static_cast<int>(mobile_apis::FunctionID::eType::OnInteriorVehicleDataID);
+
+  smart_objects::SmartObject& module_data = (*message_)
+      [application_manager::strings::msg_params][message_params::kModuleData];
+  if (module_data.keyExists(rc_rpc_plugin::message_params::kAudioControlData)) {
+    smart_objects::SmartObject& audio_control_data =
+        module_data[message_params::kAudioControlData];
+    if (audio_control_data.keyExists(message_params::kKeepContext)) {
+      audio_control_data.erase(message_params::kKeepContext);
+    }
+  }
+
+  SendNotificationToMobile(message_);
```

Comment:
I want to make sure that notifications send to all the application that subscribed to the module. Where are those logic implemented?